packages/tizen_bundle/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 0.0.1
+
+* TODO: Describe initial release.
```

Comment:
The initial version should be either 0.1.0 or 1.0.0 according to semantic versioning.
```suggestion
## 0.1.0

* Initial release.
```

Reply:
Done.